source/features/epochs/components/EpochList.scss

```diff
@@ -1,7 +1,12 @@
 .epochListContainer {
   width: 872px;
```

Comment:
Alphabetic ordering

Reply:
This is WIP pr. Need to update several components, will do this sorting once they are done.